packages/react/src/ActionList/ActionList.examples.stories.tsx

```diff
@@ -198,13 +208,11 @@ export function AsyncListWithSpinner(): JSX.Element {
         <FormControl.Label>Search branches</FormControl.Label>
         <TextInput onChange={filter} block />
       </FormControl>
-      <div role="status">
-        {results.length === 0 ? (
-          <Text sx={{display: 'block', fontSize: 1, m: 2}}>No branches match that query</Text>
-        ) : (
-          <VisuallyHidden>{results.length} branches match that query</VisuallyHidden>
-        )}
-      </div>
+      {results.length === 0 && <Text sx={{display: 'block', fontSize: 1, m: 2}}>No branches match that query</Text>}
```

Comment:
```suggestion
      {results.length === 0 ? <Text sx={{display: 'block', fontSize: 1, m: 2}}>No branches match that query</Text> : null}
```

Just a suggestion, could help with [this kind](https://eslint-react.xyz/docs/rules/no-leaked-conditional-rendering) of issue 👀

Reply:
thanks!!